### PR TITLE
EPO-2758 Update basic brightness plot to support plotting color vs Distance plot

### PR DESCRIPTION
**Pull request by georgegomez144**
Work continued from EPO-2757

### CODE REVIEW

File: src/components/charts/brightnessVsDistance/Points.jsx [hunk 1]
```diff
@@ -35,6 +35,8 @@ class Points extends React.PureComponent {
             [notActive]: (selectedData || hoveredData) && !selected && !hovered,
             [invisible]: x === null || y === null,
           });
+          const blueColorPercent = Math.floor((color / 2) * 100);
+          const redColorPercent = 100 - blueColorPercent;
```
Comment (blnkt) [inline src/components/charts/brightnessVsDistance/Points.jsx]: Great trick.  I love the results, and so does Lauren.  I asked her to think about if there are any sciency color cutoffs we should consider, or sciency hues of red and blue.  If we end up wanting to assert even more control over how to translate these values into visual colors I'm sure we can abstract the work I did in the HR Diagram to translate temps into colors (see `charts/scatter-plot/points.jsx`)

File: src/data/pages/exploring-lsst-data-1.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
       },
       "options": {
         "title": "Color Vs Distance",
+        "showFlux": true,
```
Comment (blnkt) [inline src/data/pages/exploring-lsst-data-1.json]: This is a sensible strategy to get this boolean where you need it, but I really think this is a bit of a slippery slope.  We've done a good job of abstracting the BrightnessVsDistance plot to be a GalacticPropertiesPlot but I think it's still better to rely on the x and y valueAccessors and let those inform point styling.  So you could even still use this `showFlux` flag internal to the components if you want, but I feel like `showFlux` is a little out of place in the pageData.  Instead let's just set the flag to true is the yValueAccessor is 'color'.

File: src/components/charts/brightnessVsDistance/Points.jsx (hunk 1 above)
Comment (georgegomez144) [reply src/components/charts/brightnessVsDistance/Points.jsx]: Thank you!!! I was more than excited when I put this together. Very happy with the results.

File: src/data/pages/exploring-lsst-data-1.json (hunk 2 above)
Comment (georgegomez144) [reply src/data/pages/exploring-lsst-data-1.json]: Okay, removed showFlux from the data and all places it 'touched' on it's way into the Points.jsx

All should be good. I've restarted and cleaned the gatsby to check for error with none found.

**Pull request merged**